### ISSUE TITLE
test test

### ISSUE THREAD

**Issue by carmocca** [carmocca/pytorch-lightning#2]
This issue is used by [lightning-probot](https://github.com/PyTorchLightning/probot) to manage subscriptions to labels. To subscribe yourself to a label, add a line `* label @yourusername`, or add your username to an existing line (space separated) in the body of this issue. **DO NOT COMMENT, COMMENTS ARE NOT RESPECTED BY THE BOT**.

This is a copy of https://github.com/pytorch/pytorch/issues/24422.

As a courtesy to others, please do not edit the subscriptions of users who are not you.

---

### PR status
* ready
* has conflicts
* won’t fix

### Issue/PR category
* design
* breaking change
* bug
* deprecation
* discussion
* docs
* feature
* question
* refactor
* release

### Issue/PR severity
* p0 @carmocca
* p1
* p2

### Issue/PR metadata
* duplicate
* good first issue
* help wanted
* admin
* let’s do it!
* waiting on author
* working as intended
* 3rd party
* experimental

### Code section
* accelerator: cpu
* accelerator: gpu
* accelerator: ipu
* accelerator: tpu
* argparse
* callback
* callback: device stats
* callback: early stopping
* callback: finetuning
* callback: gradient accumulation
* callback: lambda function
* callback: lr monitor
* callback: model checkpoint
* callback: model summary
* callback: prediction writer
* callback: pruning
* callback: quantization
* callback: swa
* callback: timer
* environment
* environment: kubeflow
* environment: lightning
* environment: lsf
* environment: slurm
* environment: torchelastic
* hooks
* io
* lightningcli
* lightningdatamodule
* lightninglite
* lightningmodule
* logger
* logger: comet
* logger: csv
* logger: mlflow
* logger: neptune
* logger: tensorboard
* logger: testtube (deprecated)
* logger: wandb
* logging
* loops
* lr scheduler
* optimizer
* plugin
* precision: apex
* precision: double
* precision: native amp
* profiler
* progress bar: rich
* progress bar: tqdm
* progress tracking
* strategy: ddp
* strategy: ddp spawn
* strategy: ddp2
* strategy: deepspeed
* strategy: dp
* strategy: fsdp
* strategy: horovod
* strategy: sharded
* tests
* trainer
* trainer: connector
* trainer: fit
* trainer: flag
* trainer: predict
* trainer: test
* trainer: tune
* trainer: validate

### Feature
* fault tolerance
* checkpointing
* data handling
* distributed
* optimization
* reproducibility
